### ISSUE TITLE
iOS Apple no playback in browsers

### ISSUE THREAD

**Issue by stullekovski** [YunoHost-Apps/funkwhale_ynh#261]
**How to post a meaningful bug report**
1. *Read this whole template first.*
2. *Determine if you are on the right place:*
   - *If you were performing an action on the app from the webadmin or the CLI (install, update, backup, restore, change_url...), you are on the right place!*
   - *Otherwise, the issue may be due to the app itself. Refer to its documentation or repository for help.*
   - *When in doubt, post here and we will figure it out together.*
3. *Delete the italic comments as you write over them below, and remove this guide.*
--- 

### Describe the bug

Hi all,

I’m wondering why the playback on iOS devices does not work. But on PC and Android.

Any idea how to fix the playback on iOS?
off: Where are the songs stored anyhow? Can’t find in “music” nor a “library”?

Thanks a lot!

### Context

- Hardware: root server
- YunoHost version: YunoHost 11.2.9.1 (stable) from .iso
- I have access to my server: ssh, web gui*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no

### Steps to reproduce

- open iOS browser (FF Klar, Brave, Safari), go to my pod, hit play, silence.. sometimes skipping to nothing or "connection issue"

### Expected behavior
Hear good punk music.

### Logs

from the nginx access.log:
[23/Jan/2024:19:09:07 +0100] "GET /api/v1/listen/93302781-aacd-4780-b2a7-2d868fbc5ffc/?upload=d51faf0e-95a7-4d3b-88a5-bc66764403b5 HTTP/2.0" 206 1277368 "https://******.***/library/albums/2" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1 Safari/605.1.15"

Nothing in error.log

*If applicable and useful, add screenshots to help explain your problem.*

On the song page I can see:

    “the track can not be loaded you may have connection issue”

**Comment by stullekovski** [comment]
any idea? Still can't find the problem. :/
Thanks

**Comment by stilobique** [comment]
Hum, have you look in the official repository ? To me, it's an issue with the application, not around yunohost.

All Funkwhale log can't return any issue ? (into service > funkwhale * )

**Comment by Thovi98** [comment]
I agree with @stilobique and refer to your post on the [forum](https://forum.funkwhale.audio/d/446-ios-apple-no-playback-in-browsers/2).

Please answer to that post before, because Funkwhale’s team will much more be able to help you :)

Don’t forget to tell me more about the files you are trying to play back

**Comment by stullekovski** [comment]
Thanks a lot for your replies.
I'll write over there again. 

Thought it could be Yunohost related, because some random tested Funkwhale instances were working on iOS (without Younohost backend). But as in the forum linked, it looks like a global bug for others too. Will keep an eye on tips or fixes.
Best!
